moncli/types.py

```diff
@@ -187,6 +187,66 @@ def _export(self, value):
         return { 'item_ids': [data for data in value]}
 
 
+class DropdownType(MondayType):
+    native_type = list
+    native_default = []
+    null_value = {}
+    element_type = str
+
+    def __init__(self, id: str = None, title: str = None, as_enum: Enum = None, *args, **kwargs):
+        if as_enum:
+            self.element_type = as_enum
+        super().__init__(id=id, title=title, *args, **kwargs)
+
+    def _process(self, value):
```

Comment:
This way does work.  However this other way is much cleaner and takes advantage of the assumptions defined in the requirements docs.

```python
    def _process(self, value):
        labels = {}
        return_list = []
        
        [labels.__setitem__(data['id'], data['name']) for data in self.metadata['labels']]
        for val in value:
            if isinstance(self.element_type, EnumMeta) and isinstance(val, self.element_type):
                return_list.append(val)
                continue
            try:
                label = labels[int(val)]
            except KeyError:
                raise ConversionError('Value "{}" is not a valid dropdown index.'.format(val))
            except ValueError:
                label = str(val)
                if not (label in labels.values()):
                    raise ConversionError('Value "{}" is not a valid dropdown label.'.format(val))
            if isinstance(self.element_type, EnumMeta):
                return_list.append(self.element_type(label))
            else:
                return_list.append(label)
        return return_list
```